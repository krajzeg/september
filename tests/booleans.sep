print("Should print true, false, true, false.")
print(True && True, True && False, False || True, False || False)

print("Should avoid exception via operator short-circuiting.")
print(False && its_a_trap, True || its_a_trap)
