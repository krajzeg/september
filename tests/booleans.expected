Should print true, false, true, false.
<True> <False> <True> <False>
Should avoid exception via operator short-circuiting.
<False> <True>
